weaver/wps_restapi/swagger_definitions.py

```diff
@@ -466,26 +466,78 @@ class LandingPage(ExtendedMappingSchema):
     links = LinkList()
 
 
+class FormatMimeType(ExtendedMappingSchema):
+    """
+    Used to respect ``mimeType`` field to work with pre-existing processes
+
+    .. seealso::
+        - :class:`FormatSelection`
+        - :class:`DeployFormatDefaultMimeType`
+    """
+    title = "Format"
```

Comment:
must be removed

This will cause a conflict with below `Format` schema in generate schema definitions.